client/js/components/auth/login.js

```diff
@@ -0,0 +1,21 @@
+const html = require('choo/html')
+const form = require('get-form-data')
+
+const login = (state, prev, send) => {
+  function onSubmit (e) {
+    var data = form(e.target)
+    send('user:login', data)
+    e.preventDefault()
+    return false
```

Comment:
the `return false` afaik is only necessary for older IEs, in which our tech stack wouldn't work anyways, right?